images/vnc/Dockerfile.ubuntu

```diff
@@ -57,9 +57,9 @@ ARG HOME=/home/coder
 ARG VNC_ROOT_DIR=/opt/vnc
 
 RUN cd /tmp \
-  && curl -fsSL -O https://svwh.dl.sourceforge.net/project/turbovnc/${TURBOVNC_VERSION}/turbovnc_${TURBOVNC_VERSION}_amd64.deb \
-    -O https://svwh.dl.sourceforge.net/project/libjpeg-turbo/${LIBJPEG_VERSION}/libjpeg-turbo-official_${LIBJPEG_VERSION}_amd64.deb \
-    -O https://svwh.dl.sourceforge.net/project/virtualgl/${VIRTUALGL_VERSION}/virtualgl_${VIRTUALGL_VERSION}_amd64.deb \
+  && curl -fsSL -O https://netix.dl.sourceforge.net/project/turbovnc/${TURBOVNC_VERSION}/turbovnc_${TURBOVNC_VERSION}_amd64.deb \
```

Comment:
ah hard-coding the name of a mirror does not seem like a good idea lol

Reply:
I'll play around with this: https://unix.stackexchange.com/a/86974

Reply:
no worries, this is fine to do for now. we should consider using packages from Ubuntu whenever possible though